enterprise/integrations/utils.py

```diff
@@ -57,7 +57,10 @@
 )
 
 
-OPENHANDS_RESOLVER_TEMPLATES_DIR = 'openhands/integrations/templates/resolver/'
+OPENHANDS_RESOLVER_TEMPLATES_DIR = (
```

Comment:
This is for running the resolver locally. In the packaged docker image the SAAS and local code both exist at the root directory.

Running locally without the docker image, the SAAS code is in the enterprise directory which requires setting  `OPENHANDS_RESOLVER_TEMPLATES_DIR` to `../openhands/integrations/templates/resolver/`

I've created an env so this override can be done